docs/source/getting-started/institutional-faq.md

```diff
@@ -0,0 +1,246 @@
+# Institutional FAQ
+
+This page contains common questions from users of JupyterHub,
+broken down by their roles within organizations.
+
+# For all
+
+## Is appropriate for adoption within a larger institutional context?
+
+Yes! JupyterHub has been used at-scale for large pools of users, as well
+as complex and high-performance computing. For example, UC Berkeley uses
+JupyterHub for its Data Science Education Program courses (serving over
+3,000 students). The Pangeo project uses JupyterHub to provide access
+to scalable cloud computing with Dask. JupyterHub is stable customizable
+to the use-cases of large organizations.
+
+## I keep hearing about Jupyter Notebook, JupyterLab, and now JupyterHub. What’s the difference?
+
+Here is a quick breakdown of these three tools:
+
+* **The Jupyter Notebook** is a document specification (the `.ipynb`) file that interweaves
+  narrative text with code cells and their outputs. It is also a graphical interface
+  that allows users to edit these documents
+* **JupyterLab** is a flexible and extendible user interface for interactive computing. It
+  has several extensions that are tailored for using Jupyter Notebooks, as well as extensions
+  for other parts of the data science stack.
+* **JupyterHub** is an application that can manage **multiple users** with interactive computing
+  sessions, as well as connect with infrastructure those users wish to access. It can provide
+  remote access to Jupyter Notebooks and Jupyter Lab for many people, and can connect them with
+  other compute infrastructure.
+
+# For management
+
+## Briefly, what problem does JupyterHub solve for us?
+
+JupyterHub provides a shared platform for data science and collaboration.
+It allows users to utilize familiar data science workflows (such as the scientific python stack,
+the R tidyverse, and Jupyter Notebooks) on institutional infrastructure. It also allows administrators
+some control over access to resources, security, authentication, and user identity.
+
+## Is JupyterHub mature? Why should we trust it?
+
+Yes - the core JupyterHub application recently
+reached 1.0 status, and is considered stable and performant for most institutions.
+JupyterHub has also been deployed (along with other tools) to work on
+scalable infrastructure, large datasets, and high-performance computing.
```

Comment:
How about mentioning some of the deployments at organisations people might recognise? For example Quantopian used to use JupyterHub "at scale with strict SLAs", mybinder.org uses JupyterHub in 24/7 operations, NASA/NCAR/NOAA must have deployments, CERN runs several hubs. This way we are saying "these big names you know trust JHub and are happy users, so you don't need to take our word for it". To get the point across that it is mature and reliable.

We could then finish it off with "as well as thousands(?) of small scale hubs around the world something-something' to get the point across that it is easy to setup.

Reply:
Can we brainstorm some organizations?

* UC Berkeley
* Harvard
* NASA
* NCAR
* NOAA
* Pangeo Project
* Quantopian
* NERSC
* Brookhaven National Lab
* Capital One

Maybe @rcthomas or @rpwagner can recommend others in the HPC/science infra space?

Reply:
* ALCF
* University of Chicago
* Minnesota Supercomputing Institute
* Globus
* Plus whatever is in the [Gallery of JupyterHub Deployments](https://jupyterhub.readthedocs.io/en/stable/gallery-jhub-deployments.html)

Although, I think a representative sample based on scale, use case, and implementation would be better than an extensive list.

Reply:
CERN, University of Oslo, NTNU, University of Sheffield, Université Paris Sud,  University of Versailles, MathHub

Not sure how many are appropriate

Reply:
Lawrence Livermore National Laboratory

Reply:
I'd pick out maybe three-five(?) of the names with the most name recognition as serious (as in they have challenging needs and wouldn't be happy if it was constantly broken) tech places.

Without knowing what kind of deployment(s) they have I'd pick as places I think are recognised the world over: Harvard, NASA, CERN, Capital One, Bloomberg.

Reply:
The way it's written now, I'm breaking it down be "type of organization". You think it's better to just pick 5 or 6?

Reply:
I'd love to see them all written down! =D I'm very happy to have seen this list, it makes me as a contributor to JupyterHub very happy!

Reply:
I like the broken down list! Hadn't noticed it